Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -695,15 +695,19 @@ - (UITableViewCell *)cellForArchivedConversationsRow:(UITableView *)tableView
     UIStackView *stackView = [UIStackView new];
     stackView.axis = UILayoutConstraintAxisHorizontal;
     stackView.spacing = 5;
+    // If alignment isn't set, UIStackView uses the height of
+    // disclosureImageView, even if label has a higher desired height.
+    stackView.alignment = UIStackViewAlignmentCenter;
```

Comment:
I'm still getting a feel for `UIStackView`, but this was very surprising.  I'm probably just going to get in the habit of always setting alignment to center (unless we need otherwise) to avoid this class of issue.  I don't understand why the default behavior is what it is.

Reply:
Yeah, that doesn't sound right. Are you sure that the stackView has enough space to grow vertically?

If we're pinned to the container, and the container isn't large enough, somethings got to give. And based on the hugging/compression configuration we're using that's going to be the label, not the image.